routes/web.php

```diff
@@ -471,13 +467,23 @@
 
     // Users
     $router->group([
+        'namespace' => 'Users',
         'prefix' => 'users',
         'middleware' => ['scope:users']
     ], function () use ($router) {
         // Public access
         $router->get('/', 'UsersController@index');
         $router->get('/{id:[0-9]+}', 'UsersController@show');
 
+        // Sub-user routes
```

Comment:
Why is this here, but repeated in features routes?

Reply:
I checked I don't see it repeated but I was wondering why is it outside auth?

Reply:
Users are available to anonymous users to provide the users name for post detail - but we filter out email and other sensitive info.